packages/driver/src/cy/commands/cookies.ts

```diff
@@ -79,28 +79,16 @@ function cookieValidatesSecurePrefix (options) {
   return options.secure === false
 }
 
-interface InternalGetCookieOptions extends Partial<Cypress.Loggable & Cypress.Timeoutable> {
-  _log?: Log
-  cookie?: Cypress.Cookie
-}
-
-interface InternalGetCookiesOptions extends Partial<Cypress.Loggable & Cypress.Timeoutable> {
-  _log?: Log
-  cookies?: Cypress.Cookie[]
-}
-
-interface InternalSetCookieOptions extends Partial<Cypress.SetCookieOptions> {
-  _log?: Log
-  name: string
-  cookie?: Cypress.Cookie
-}
-
-type InternalClearCookieOptions = InternalGetCookieOptions
-
-interface InternalClearCookiesOptions extends Partial<Cypress.Loggable & Cypress.Timeoutable> {
-  _log?: Log
-  cookies?: Cypress.Cookie[]
-  domain?: any
+function validateDomainOption (domain: any, commandName: string, log: Log | undefined) {
+  if (domain != null && !_.isString(domain)) {
```

Comment:
might make the comparison clearer at first glance?
```suggestion
  if (domain !== null && domain !== undefined && !_.isString(domain)) {
```

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/24471/commits/89f51cc73224701fc0cdc190fdadbf5290fd1d56